docs/pages/versions/unversioned/guides/adhoc-builds.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Installing a Custom iOS Expo Client
+---
+
+![Build Overview](/static/images/adhoc-builds-overview.gif)
+
+Build and install a custom version of the Expo Client (link) with your own Apple Credentials using our build service. This custom version of the Expo Client contains features that were previously only available on the Android versions. Our build service will prepare your custom Expo Client, and you can install it to your iOS device directly from our website.
+
+## 0. Prerequisites
+
+You'll need an [Apple Developer Account](https://developer.apple.com/programs)
+
+## 1. Install Expo CLI
+
+Expo CLI is the tool for developing and building Expo apps. Run `npm install -g expo-cli` (or `yarn global add expo-cli`) to get it.
+
+If you haven't created an Expo account before, you'll be asked to create one when running the build command.
+
+**Windows users** must have WSL enabled. We recommend picking Ubuntu from the Windows Store. Be sure
+to Launch Ubuntu at least once. After that, use an Admin powershell to run:
+`Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-Linux`
+
+## 2. Start the build
+
+Run `expo client:ios`.
+
+### 2a. Provide Apple Credentials
+
+You are given a choice of letting the `expo` client create the
+necessary credentials for you, while still having a chance to provide
+your own overrides. Your Apple ID and password is used locally and
+never saved on Expo's servers.
+
+```bash
+$ expo client:ios
+[16:44:37] Checking if current build exists...
+
+[16:44:37] No currently active or previous builds for this project.
+[16:44:37]
+Please enter your Apple Developer Program account credentials. These credentials are needed to manage certificates, keys and provisioning profiles in your Apple Developer account.
+
+The password is only used to authenticate with Apple and never stored.
+
+? What\'s your Apple ID? xxx@yyy.zzz
+? Password? [hidden]
+✔ Authenticated with Apple Developer Portal successfully!
+[16:44:46] You have 4 teams associated with your account
+? Which team would you like to use? 3) ABCDEFGHIJ "John Turtle" (Individual)
+
+? Select an iOS distribution certificate to use for code signing:
+❯ [Create a new certificate]
+  [Upload an existing certificate]
+
+? Select an authentication token signing key to use for push notifications: (Use arrow keys)
+❯ Used in apps: @john-turtle/expo-client-home-xxx (Key ID: XXX)
+  [Create a new key]
+  [Upload an existing key]
+```
+
+We ask you if you'd like us to handle your Distribution Certificate or
+use your own. If you have previously used `expo-cli` for building a standalone app
```

Comment:
looks like newlines are inserted at arbitrary points in this paragraph, maybe inserted by text editor?

Reply:
i've gone and removed these newlines